src/Console/ConsoleOptionParser.php

```diff
@@ -169,7 +169,7 @@ public function __construct($command = null, $defaultOptions = true)
      *
      * @param string|null $command The command name this parser is for. The command name is used for generating help.
      * @param bool $defaultOptions Whether you want the verbose and quiet options set.
-     * @return self
+     * @return $this
      */
     public static function create($command, $defaultOptions = true)
```

Comment:
Careful, you should check the original PR from back then, and check out the commit right before it and see which of the "self" have been there before - like this one. Those should still not be changed, as they have been correct.
Mainly static methods, but also the ones that return a different instance (not chaining).